test/test_excerpt.rb

```diff
@@ -1,22 +1,23 @@
-require 'helper'
+require "helper"
 
 class TestExcerpt < JekyllUnitTest
   def setup_post(file)
-    Document.new(@site.in_source_dir(File.join('_posts', file)), {
-      site:       @site,
-      collection: @site.posts
+    Document.new(@site.in_source_dir(File.join("_posts", file)), {
+      :site => @site,
+      :collection => @site.posts
     }).tap(&:read)
   end
 
   def do_render(document)
-    @site.layouts = { "default" => Layout.new(@site, source_dir('_layouts'), "simple.html")}
+    @site.layouts = { "default" => Layout.new(@site, source_dir("_layouts"),
+                                                                "simple.html") }
```

Comment:
I'd move the key down instead of the one argument:

```ruby
@site.layouts = {
  "default" => Layout.new(@site, source_dir("_layouts"), "simple.html")
}
```